cmd/protoc-gen-go-grpc/go.mod

```diff
@@ -1,6 +1,6 @@
 module google.golang.org/grpc/cmd/protoc-gen-go-grpc
 
-go 1.22
+go 1.22.8
```

Comment:
We got pushback from users when we specified a min Go minor version in our go.mod. It forced all users to bump their min versions to use the latest version of gRPC Go. So we decided to remove the minor version in https://github.com/grpc/grpc-go/pull/7831. Let me open an issue with go-control-plane to see if they are willing to drop the minor version.

Reply:
I've opened Opened https://github.com/envoyproxy/go-control-plane/issues/1077, let's see what they think.